manifests/0000_12_etcd-operator_01_operator.cr.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   annotations:
     release.openshift.io/create-only: "true"
 spec:
-  managementState: Unmanaged
+  managementState: Managed
```

Comment:
we want this Unmanaged right?

Reply:
for now yes. I was over-optimistic in bringing it in all at the same time :)